scipy/stats/tests/test_continuous.py

```diff
@@ -1018,34 +1018,28 @@ def test_make_distribution(self, i, distdata):
         distname = distdata[0]
 
         slow = {'argus', 'exponpow', 'exponweib', 'genexpon', 'gompertz', 'halfgennorm',
-                'johnsonsb', 'kstwobign', 'powerlognorm', 'powernorm', 'recipinvgauss',
-                'vonmises_line'}
+                'johnsonsb', 'kappa4', 'ksone', 'kstwo', 'kstwobign', 'powerlognorm',
+                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line'}
         if not int(os.environ.get('SCIPY_XSLOW', '0')) and distname in slow:
             pytest.skip('Skipping as XSLOW')
 
-
         if distname in {  # skip these distributions
-            'genpareto', 'genextreme', 'genhalflogistic',  # complicated support
-            'kstwo', 'kappa4', 'tukeylambda',  # complicated support
-            'levy_stable',  # levy_stable does things differently...
-            'ksone',  # tolerance issues
-            'norminvgauss',  # private methods seem to have broadcasting issues
+            'levy_stable',  # private methods seem to require >= 1d args
```

Comment:
I'll let someone else handle this. I tried a little, but I didn't want to get sucked into an endless game of whack-a-mole.